### PR TITLE
first prototype with gaiax pipeline

### PR DESCRIPTION
**Pull request by gorkazar7**
gaiax pipe structure based on airflow

### CODE REVIEW

File: devops/.env [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Meta-Database
+POSTGRES_USER=airflow
+POSTGRES_PASSWORD=airflow
+POSTGRES_DB=airflow
+
+# Airflow Core
+AIRFLOW__CORE__FERNET_KEY=UKMzEm3yIuFYEq1y3-2FxPNWSVwRASpahmQ9kQfEr8E=
```
Comment (garloff) [inline devops/.env:7]: Hopefully no valuable secret?

Comment (gorkazar7) [reply devops/.env:7]: no use at the moment

Comment (garloff) [reply devops/.env:7]: Great, just being careful :-)

**Review by garloff — APPROVED** [review]
I'd still like to see some doc or pointer in README.md, but this does not block the merging from my perspective (as our time window in the Hackathon is limited.)
So feel free to
[  ] add more commits with some documentation (preferred option)
[  ] hit the merge button even without it

**Pull request merged**